src/templates/atoms.njk

```diff
@@ -1315,31 +1323,19 @@
                     <div class="a-input__wrapper">
                         <input type="text" name="text-field-success" id="text-field-success" placeholder="Placeholder" />
                     </div>
-                </div>
-            </div>
-            <div>
-                <pre class="a-pre a-pre--scrollable"><code class="html">&lt;div class="a-input has-success"&gt;
-  &lt;label class="a-input__label" for="text-field-success"&gt;Success text field&lt;/label&gt;
-  &lt;div class="a-input__wrapper"&gt;
-    &lt;input type="text" name="text-field-success" id="text-field-success" placeholder="Placeholder" /&gt;
-  &lt;/div&gt;
-&lt;/div&gt;</code></pre>
-            </div>
-        </div>
-        <div class="d">
-            <div>
-                <div class="a-input has-warning">
-                    <label class="a-input__label" for="text-field-warning">Warning text field</label>
-                    <div class="a-input__wrapper">
-                        <input type="text" name="text-field-warning" id="text-field-warning" placeholder="Placeholder" />
+                    <div class="a-input__info">
+                        <small class="a-input__success-text">Success description</small>
```

Comment:
Persoonlijk zou ik hier het fysieke icoon gebruiken i.p.v. de "in de code verstopte" versie.